src/tink/http/Header.hx

```diff
@@ -91,12 +91,23 @@ class Header {
     
   public function concat(fields:Array<HeaderField>):Header
     return new Header(this.fields.concat(fields));
+    
+  public function getContentLength()
+    return switch byName(CONTENT_LENGTH) {
+      case Success(Std.parseInt(_) => null): Failure(new Error(UnprocessableEntity, 'Invalid Content-Length Header'));
+      case Success(Std.parseInt(_) => v): Success(v);
+      case Failure(e) if(e.message == headerNotFound(CONTENT_LENGTH)): Success(0); // this should be ok
```

Comment:
You are right. The thing is, I don't want people to write something like this: 

```haxe
switch req.getContentLength() {
  case Success(length): rawBody = rawBody.limit(length);
  default:
}
```

That will just cut off the body, if it is chunked. I think it would be better to generate an error with [411](https://httpstatuses.com/411). It kind of depends on how you're going to use it. For requests without bodies I'm fine with having a size of 0, but for request with bodies of unbounded size (which is what chunked basically means) 0 is not the best value.